host-configs/TOTAL/pecan.cmake

```diff
@@ -0,0 +1,47 @@
+set(COMPILER_HOME "/apps/gcc/8.2.0/x86_64")
+set(MPI_HOME "/hrtc/apps/mpi/openmpi/4.0.1/RDHPC/gcc/8.2.0")
+
+set(CMAKE_C_COMPILER ${COMPILER_HOME}/bin/gcc CACHE PATH "" FORCE)
+set(CMAKE_CXX_COMPILER ${COMPILER_HOME}/bin/g++ CACHE PATH "" FORCE)
+set(ENABLE_FORTRAN OFF CACHE BOOL "" FORCE)
+
+set(ENABLE_MPI ON CACHE PATH "" FORCE)
+set(ENABLE_OPENMP ON CACHE PATH "" FORCE)
+
+set(MPI_C_COMPILER "${MPI_HOME}/bin/mpicc" CACHE PATH "" FORCE)
+set(MPI_CXX_COMPILER "${MPI_HOME}/bin/mpicxx" CACHE PATH "" FORCE)
+set(MPIEXEC_EXECUTABLE "${MPI_HOME}/bin/mpirun" CACHE PATH "" FORCE)
+#set(MPIEXEC_NUMPROC_FLAG   "-p pecan -n" CACHE STRING "")
+set(ENABLE_WRAP_ALL_TESTS_WITH_MPIEXEC ON CACHE BOOL "")
+
+set(ENABLE_CUDA ON CACHE PATH "" FORCE)
+if(ENABLE_CUDA)
+  set(CUDA_TOOLKIT_ROOT_DIR /hrtc/apps/cuda/10.2.89/x86_64 CACHE PATH "")
+  set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER} CACHE STRING "")
+  set(CMAKE_CUDA_COMPILER ${CUDA_TOOLKIT_ROOT_DIR}/bin/nvcc CACHE STRING "")
+  set(CUDA_ARCH sm_75 CACHE STRING "")
+  set(CMAKE_CUDA_STANDARD 14 CACHE STRING "")
+  ### The inclusion of -std=c++14 is a workaround for a cuda10/gcc8 bug ###
+  set(CMAKE_CUDA_FLAGS "-restrict -arch ${CUDA_ARCH} --expt-relaxed-constexpr --expt-extended-lambda -Werror cross-execution-space-call,reorder,deprecated-declarations -Xcompiler -std=c++14" CACHE STRING "")
+  set(CMAKE_CUDA_FLAGS_RELEASE "-O3 -DNDEBUG -Xcompiler -DNDEBUG -Xcompiler -O3" CACHE STRING "")
+  set(CMAKE_CUDA_FLAGS_RELWITHDEBINFO "-g -lineinfo ${CMAKE_CUDA_FLAGS_RELEASE}" CACHE STRING "")
+  set(CMAKE_CUDA_FLAGS_DEBUG "-g -G -O0 -Xcompiler -O0" CACHE STRING "")
+endif()
```

Comment:
ENABLE_HYPRE_CUDA ??

Reply:
What do you mean? I do not think that I understand.
Do you think that I should activate `HYPRE_CUDA` by default?

Reply:
yes

Reply:
I've tested the compilation of `hypre` with `cuda` but it's not working on pecan...
But a more recent version of `hypre` is working.
I would like to have the pecan build like this and deal with the GPU stuff in another PR (maybe wait for `GEOSX` to update `hypre`).

Is it OK for you?